core/impuestovehicular-ejb/src/main/java/us/codezen/payment/dto/CalculateLatePaymentReqDTO.java

```diff
@@ -0,0 +1,17 @@
+package us.codezen.payment.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Setter
+@Getter
+
```

Comment:
puedes agregar` @Data` en cambio  dle setter and getter

Reply:
[099df8c](https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular/pull/3/commits/099df8c6a461b19ebf7c2dcffcce9b9f9a961eab)